test/specs/modules/Popup/Popup-test.js

```diff
@@ -324,6 +324,14 @@ describe('Popup', () => {
     })
   })
 
+  describe('context', () => {
+    it('should set context as triggerRef if passed as prop', () => {
+      const context = {}
+      const popup = wrapperMount(<Popup context={context} />)
+      expect(popup.instance().triggerRef).to.equal(context)
+    })
+  })
```

Comment:
Our tests will be more robust if they test the public behavior.  Since triggerRef is an implementation detail, we want to avoid testing it as it won't validate that the feature works as expected by the user.  The public expectation is that the popup is positioned on the context node.

Suggest doing something like:

1. Create an element and positioning it fixed in the browser somewhere
1. Pass that element to context and open the Popup
1. Capture Popup position information and cleanup the DOM
1. Assert that the Popup was in the correct position

You can reference a positioning test here:

https://github.com/Semantic-Org/Semantic-UI-React/blob/0d00166bd55e9cdab5208678b340179ab4c548c7/test/specs/modules/Popup/Popup-test.js#L142